napari/layers/points/points.py

```diff
@@ -378,6 +379,12 @@ def __init__(
         shown=True,
         projection_mode='none',
     ) -> None:
+        settings = get_settings()
+        if edge_color is None:
+            edge_color = settings.appearance.edge_color
+        if face_color is None:
+            face_color = settings.appearance.face_color
```

Comment:
```suggestion
        if edge_color is None:
            edge_color = get_settings().appearance.edge_color
        if face_color is None:
            face_color = get_settings().appearance.face_color
```

Do not use settings if not needed.

Reply:
Gah, I had it that way but thought I was making it more efficient the other way!
Makes sense though, thanks!